client/my-sites/stats/controller.js

```diff
@@ -707,18 +713,20 @@ module.exports = {
 			);
 
 			ReactDom.render(
-				React.createElement( FollowsComponent, {
-					path: context.path,
-					sites: sites,
-					siteId: siteId,
-					page: pageNum,
-					perPage: 20,
-					total: 10,
-					followersList: followersList,
-					followType: followType,
-					followList: followList,
-					domain: siteDomain
-				} ),
+				React.createElement( ReactRedux.Provider, { store: context.store },
```

Comment:
I forgot we have [`renderWithReduxStore`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/react-helpers/index.js#L8) for this.